SYCL/Functor/kernel_functor.cpp

```diff
@@ -1,10 +1,9 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -o %t.out %s
 // RUN: cd %T
-// RUN: %HOST_RUN_PLACEHOLDER %t.out
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %RUN_ON_HOST %t.out
```

Comment:
The version of this test file on intel/llvm has just `// RUN: %RUN_ON_HOST %t.out`
So to keep both of the versions consistent, I have the exact same contents from https://github.com/intel/llvm/blob/48e81620b1219e1a5e22d4e6528ce8473afd4bf2/sycl/test/functor/kernel_functor.cpp

Please let me know if I should add CPU and GPU targets back here.

Reply:
Please preserve RUN lines from intel/llvm-test-suite version

Different run lines should be used in intel/llvm and intel/llm-test-suite